src/coreclr/jit/targetamd64.h

```diff
@@ -385,8 +398,9 @@
   // The following defines are useful for iterating a regNumber
   #define REG_FIRST                REG_EAX
   #define REG_INT_FIRST            REG_EAX
-  #define REG_INT_LAST             REG_R15
-  #define REG_INT_COUNT            (REG_INT_LAST - REG_INT_FIRST + 1)
+  #define REG_INT_LAST             REG_R23
+  #define REG_INT_LAST_APX_AWARE             get_REG_INT_LAST()
```

Comment:
no need of `REG_INT_LAST_APX_AWARE`? just use `get_REG_INT_LAST()` directly for `REG_INT_COUNT`?

Reply:
Changed